lib/shelley/src/Cardano/Wallet/Shelley/Compatibility.hs

```diff
@@ -2040,28 +2039,36 @@ instance DecodeAddress ('Testnet pm) where
 
 decodeBytes :: Text -> Either TextDecodingError ByteString
 decodeBytes t =
-    case tryBase16 t <|> tryBech32 t <|> tryBase58 t of
+    case tryBech32 t <|> tryBase58 t of
         Just bytes ->
             Right bytes
         _ ->
-            Left $ TextDecodingError
-                "Unrecognized address encoding: must be either bech32, base58 or base16"
+            Left $ TextDecodingError $ mconcat
+                [ "Unrecognized address encoding: must be either bech32 or base58."
+                , " Perhaps your address is not entirely right?"
+                , " Every character has to be correct!"
+                ]
```

Comment:
Suggestion: by using `unwords` here, rather than `mconcat`, we can avoid having to manually insert spaces.

```suggestion
            Left $ TextDecodingError $ unwords
                [ "Unrecognized address encoding: must be either bech32 or base58."
                , "Perhaps your address is not entirely right?"
                , "Every character has to be correct!"
                ]
```